.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@ repos:
     rev: 21.10b0
     hooks:
     -   id: black
-        args: [--line-length=120]
+        exclude: '^versioneer.py$|^timemachine/_version.py'
```

Comment:
Unfortunately these exclusions need to be duplicated here from `pyproject.toml` (see https://github.com/psf/black/issues/438)